### PR TITLE
Badvideo test workaround

### PR DESCRIPTION
**Pull request by cafed00d4j**

### CODE REVIEW

**Comment by cafed00d4j** [comment]
Summary of Failure for badvideotest

* badvideotest fails only on memex machine
* opencv on memex machine warns of an error related to "libdc1394"
* libdc1394 is related to the camera device drivers "/dev/raw1394"
* The VideoCapture API provided by OpenCV is used to read video feed from either a file or camera
* badvideotest/video-file is probably being recognised a camera input and thus is null when read
* since the frame being read is null, the PoT app crashes
* This workaround aims to Log this error so that we can detect such a crash and not halt the program. We can resume the program on succeeding videos

Since badvideotest/video-file is being read perfectly weel on my local machine this error seems to be a problem only on the memex machine. It is most likely due to the OpenCV libdc1394 error listed earlier. Possible steps of actions is to:
1. resolve libdc1394 error by setting
"sudo ln /dev/null /dev/raw1394"
2. Reinstall OpenCV

File: src/main/java/gov/nasa/jpl/memex/pooledtimeseries/PoT.java [hunk 1]
```diff
@@ -389,6 +401,10 @@ private static void writeSimilarityToJSONFile(ArrayList<Path> files, double[][]
 	      capture.read(original_frame);
 	
 	      if (original_frame.empty()) {
+            if(frame_index == 0) {
+                LOG.severe("Cannot read the specified video file. Either its corrupt or there's a codec missing. Exiting...");
+                System.exit(1);
```
Comment (chrismattmann) [inline src/main/java/gov/nasa/jpl/memex/pooledtimeseries/PoT.java]: why don't we simply just continue the loop here?

File: pom.xml [hunk 2]
```diff
@@ -50,19 +50,19 @@
...
-        <version>1.3</version>
+      <groupId>commons-cli</groupId>
+      <artifactId>commons-cli</artifactId>
+      <version>1.3</version>
     </dependency>	
     <dependency>
-        <groupId>com.googlecode.json-simple</groupId>
-        <artifactId>json-simple</artifactId>
-        <version>1.1.1</version>
+      <groupId>com.googlecode.json-simple</groupId>
```
Comment (chrismattmann) [inline pom.xml]: these are formatting changes - orthogonal.

File: pom.xml [hunk 3]
```diff
@@ -50,19 +50,19 @@
...
-	    <version>2.4</version>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
 	</dependency>   
     <dependency>
-        <groupId>commons-cli</groupId>
-        <artifactId>commons-cli</artifactId>
-        <version>1.3</version>
+      <groupId>commons-cli</groupId>
```
Comment (chrismattmann) [inline pom.xml]: formatting issue - we don't need to reformat this part. Please remove.

File: pom.xml [hunk 4]
```diff
@@ -50,19 +50,19 @@
       <version>2.4.9-4</version>
    </dependency>
 	<dependency>
-	    <groupId>commons-io</groupId>
-	    <artifactId>commons-io</artifactId>
-	    <version>2.4</version>
+      <groupId>commons-io</groupId>
```
Comment (chrismattmann) [inline pom.xml]: more orthogonal formatting.

Comment (chrismattmann) [inline pom.xml]: please remove, no need for this.

**Pull request closed without merging**